src/driftpy/drift_user.py

```diff
@@ -53,11 +51,22 @@ def __init__(
             self.program, self.user_public_key
         )
 
+        self.initial_data = None
+
+        self.is_subscribed = False
+
     async def subscribe(self):
+        data = await get_account_data_and_slot_with_retry(
```

Comment:
this feels redundant, why cant we just solve it at the account subscriber level?